modules/lnd.nix

```diff
@@ -45,10 +46,15 @@ in {
       description = "The data directory for LND.";
     };
     rpcPort = mkOption {
-      type = types.ints.u16;
```

Comment:
Is it necessary to change this? From the NixOS module: "types.port: A port number. This type is an alias to types.ints.u16."

If so, should we change all occurrences of `types.ints.u16` -> `types.port`

Reply:
Yes, we should change them separately: https://github.com/fort-nix/nix-bitcoin/pull/190